src/service/src/grpc.rs

```diff
@@ -337,13 +348,19 @@ where
                                 break;
                             }
                         };
+
+                        if let Ok(ts) = UNIX_EPOCH.elapsed() {
+                            state.metrics.last_command_received.set(ts.as_secs());
+                        }
```

Comment:
IIUC, `SystemTime::elapsed` returning `None` for the unix epoch would be very unexpected and means that the system clock is seriously off. Should we log an error in this case, or even panic? If we silently ignore the issue, we might misinterpret outdated metrics values as a networking issue instead of a system clock issue.

Reply:
Yeah, fair enough! I updated to log an error if it fails to get the system time. (Panicking feels a little extreme; I'd hate to take down the process over this.)